src/lib/targeting.ts

```diff
@@ -46,12 +44,13 @@ export const shouldThrottle = (
     }
 
     const viewsInThrottleWindow = views.filter(view => {
-        return daysSince(new Date(view.date), now) <= config.maxViewsDays;
+        return daysSince(new Date(view.date), now) < config.maxViewsDays;
```

Comment:
Given `maxViewsDays=30`...
We want the throttle to apply when `daysSince()` returns `30` (effectively at day 31) as it starts counting at 0 days until it's been 24 hours.